test/webhook-proxy.test.ts

```diff
@@ -18,14 +24,14 @@ describe('webhook-proxy', () => {
     beforeEach((done) => {
       app = express()
 
-      app.get('/events', sse, function (req, res) {
-        res.json({}, 'ready')
+      app.get('/events', sse, (req, res) => {
+        res.json({}, 'ready') // fixme: not sure how to convert this to TypeScript
```

Comment:
Looking at the types for `res.json()`, it's of type [`Send`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3ee3f99b0011b6541ef959868b970b256bca5460/types/express-serve-static-core/index.d.ts#L468), which takes 0-1 arguments, but we are passing 2.

I'm not quite sure which signature of `res.json()` this is and how to properly convert it to TypeScript. Help is welcome here. 💟